### ISSUE TITLE
High temperature on a MacOS

### ISSUE THREAD

**Issue by mgarcia96** [laravel/homestead#1036]
Hey there, I just bought a new MacBook Pro and when I run vagrant the CPU shots to very high levels, and the PC becomes very very hot!

## Versions
  - Vagrant: Vagrant 2.2.3
  - Provider: Virtualbox
  - Homestead: 7.0.0

## Host operating system
MacOS 14.0.2
## Homestead.yaml
```yaml
---
ip: "192.168.10.10"
memory: 1024
cpus: 1
provider: virtualbox```

**Comment by browner12** [comment]
open up your "Activity Monitor" and check the Energy Consumption of Terminal.

I was having a high energy use when I closed the clamshell on my laptop, because Virtualbox was not shutting down correctly. 

Try halting you machine and then bringing it back up, and then see if the high power use continues.

**Comment by mgarcia96** [comment]
> open up your "Activity Monitor" and check the Energy Consumption of Terminal.
> 
> I was having a high energy use when I closed the clamshell on my laptop, because Virtualbox was not shutting down correctly.
> 
> Try halting you machine and then bringing it back up, and then see if the high power use continues.

Hey! In Activity Monitor there is a vboxheadless process that consume a lot!
Yes I tried to halt, and when vagrant is powered off the temperature is normal

**Comment by mgarcia96** [comment]
Yes but, with npm run watch is a serius problem

**Comment by browner12** [comment]
yup, that's expected to have high resource usage with processes like that.

**Comment by mgarcia96** [comment]
Then this is normal? Is dangerous? My mac is new😪

**Comment by browner12** [comment]
nope, don't get hung up on that unless it's *always* running hot, even when idle.

**Comment by mgarcia96** [comment]
Hmm okei thanks!!!!

**Comment by svpernova09** [comment]
> Yes but, with npm run watch is a serius problem

You should run npm on your Mac, not *in* the VM. Otherwise you should give your VM a couple more CPU cores.

**Comment by mgarcia96** [comment]
> > Yes but, with npm run watch is a serius problem
> 
> You should run npm on your Mac, not _in_ the VM. Otherwise you should give your VM a couple more CPU cores.

I will try that, thanks!!!

**Comment by browner12** [comment]
Unless NPM has squared this stuff up, I'd actually advise against running in the host.  I've had issues in the past where the installed dependencies were different depending on if I ran in the host or the guest.

Hopefully that's not an issue anymore, and you can do it either place.
